src/utils/netlify-utils.ts

```diff
@@ -0,0 +1,95 @@
+import axios from "axios"
+
+import { config } from "@config/config"
+
+import logger from "@root/logger/logger"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+const NETLIFY_ACCESS_TOKEN = config.get("netlify.accessToken")
+
+type NetlifySiteDetails = {
+  id: string
+  password: string
+  build_settings: {
+    repo_path: string
+    repo_branch: string
+  }
+}
+
+type UpdateNetlifySiteDetails = {
+  password: string
+  build_settings: {
+    stop_builds: boolean
+  }
+}
+
+// Retrieves all netlify sites which build from the given repo's staging branch
+const getNetlifySiteDetails = async (repoName: string) => {
+  const endpoint = "https://api.netlify.com/api/v1/sites"
+  const headers = {
+    Authorization: `Bearer ${NETLIFY_ACCESS_TOKEN}`,
+    "Content-Type": "application/json",
+  }
+  const repoPath = `${GITHUB_ORG_NAME}/${repoName}`
+  try {
+    const allSites = await axios.get<NetlifySiteDetails[]>(endpoint, {
+      headers,
+    })
+    const relatedSites = allSites.data.filter(
+      (site) =>
+        site.build_settings.repo_path === repoPath &&
+        site.build_settings.repo_branch === "staging"
+    )
+    return relatedSites
+  } catch (err: any) {
```

Comment:
nit: `err: unknown`

Reply:
Changed in https://github.com/isomerpages/isomercms-backend/pull/828/commits/66f5014956123616a85c3afe4f7f897221555d69